### PR TITLE
chore(console): remove type errors

### PR DESCRIPTION
**Pull request by eyllanesc**

### CODE REVIEW

**Review by branchvincent — DISMISSED** [review]
@eyllanesc thanks for your contribution! This looks good to me. We'll just need to wait for another maintainer to approve/merge

**Review by neersighted — APPROVED** [review]

**Comment by dimbleby** [comment]
#5279

**Review by branchvincent — APPROVED** [review]
thanks @eyllanesc! I think we can handle running mypy in the proper environment separately

**Pull request merged**
